db/app.js

```diff
@@ -93,10 +96,15 @@ app.use((err, req, res, next) => {
 });
 
 app.use((err, req, res, next) => {
-  console.log(err);
   if (err.message === "Article not found.") {
     res.status(404).send(err);
   } else next(err);
 });
 
+app.use((err, req, res, next) => {
+  if (err.message === "Comment not found.") {
+    res.status(404).send(err);
+  } else next(err);
+});
+
```

Comment:
You should start combining these error handlers, if you included a statusCode in your promise rejects then you could have a single error handler that sent res.status(err.statusCode).send(err.message). It would work every time! 

Similiar with the psql codes you could put them all into one error handler with different if statements